ONSdigital/ssdc-rm-caseprocessor
# 2610 audit trail everything in the rm events table should have a created by

# Motivation and Context

We want to keep an audit trail of all the actions made against cases by a user. I've added in a created by column in the DB to add the user in from the event.

# What has changed



- Added createdBy column in DB and EventDTO
# How to test?



- Run with the Support tool branch on the trello ticket
- create a survey/collex and upload a sample
- Create some refusals/ invalid addresses/ fulfilments/deactivate uacs in the support tool for the case and the user should be specified in the Events table in the DB

# Links



[Trello](https://trello.com/c/qfJKp8m3)
